### ISSUE TITLE
Running benchmarks fails with 404 on favicon.ico

### ISSUE THREAD

**Issue by hterkelsen** [angular/angular#1463]
Running a benchmark with 'Google Chrome 42.0.2311.90' is failing with a 404 on favicon.ico

```
Dart SDK detected
[11:45:31] Using gulpfile /tmp/runnes/angular_resources/angular/gulpfile.js
[11:45:31] Starting 'serve.js.dart2js'...
[11:45:31] Finished 'serve.js.dart2js' after 114 ms
[11:45:31] Server started on 8002 port
connect deprecated res.headerSent: use standard res.headersSent node_modules/gulp-connect/node_modules/connect/lib/proto.js:127:22
Testing against Dartium
Starting selenium standalone server...
[launcher] Running 1 instances of WebDriver
Selenium standalone server started at http://172.18.76.161:48017/wd/hub
Started
BENCHMARK ng2.naive_infinite_scroll
Description:
- appSize: 1
- forceGc: false
- iterationCount: 20
- lang: dart
- regressionSlopeMetric: scriptTime
- runId: 70a9aad0-e78d-11e4-815f-9b30cb56bc49
- sampleSize: 20
- scrollIncrement: 40
- userAgent: Mozilla/5.0 (X11; Linux x86_64) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/42.0.2311.90 Safari/537.36
Metrics:
- gcAmount: gc amount in kbytes
- gcTime: gc time in and ouside of script in ms
- majorGcTime: time of major gcs in ms
- pureScriptTime: script execution time in ms, without gc nor render
- renderTime: render time in and ouside of script in ms
- scriptTime: script execution time in ms, including gc and render

          gcAmount |             gcTime |        majorGcTime |     pureScriptTime |         renderTime |         scriptTime
------------------ | ------------------ | ------------------ | ------------------ | ------------------ | ------------------
              0.00 |               0.00 |               0.00 |             200.35 |             128.59 |             204.06
>> http://localhost:8002/favicon.ico 0:0 Failed to load resource: the server responded with a status of 404 (Not Found)
FBENCHMARK ng2.naive_infinite_scroll
Description:
- appSize: 2
- forceGc: false
- iterationCount: 20
- lang: dart
- regressionSlopeMetric: scriptTime
- runId: 70a9aad0-e78d-11e4-815f-9b30cb56bc49
- sampleSize: 20
- scrollIncrement: 40
- userAgent: Mozilla/5.0 (X11; Linux x86_64) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/42.0.2311.90 Safari/537.36
Metrics:
- gcAmount: gc amount in kbytes
- gcTime: gc time in and ouside of script in ms
- majorGcTime: time of major gcs in ms
- pureScriptTime: script execution time in ms, without gc nor render
- renderTime: render time in and ouside of script in ms
- scriptTime: script execution time in ms, including gc and render

          gcAmount |             gcTime |        majorGcTime |     pureScriptTime |         renderTime |         scriptTime
------------------ | ------------------ | ------------------ | ------------------ | ------------------ | ------------------
           3717.24 |              12.18 |               0.00 |             194.74 |             132.54 |             198.18
>> http://localhost:8002/favicon.ico 0:0 Failed to load resource: the server responded with a status of 404 (Not Found)
          14022.40 |              18.35 |              18.35 |             209.15 |             139.99 |             209.38
FBENCHMARK ng2.naive_infinite_scroll
Description:
- appSize: 4
- forceGc: false
- iterationCount: 20
- lang: dart
- regressionSlopeMetric: scriptTime
- runId: 70a9aad0-e78d-11e4-815f-9b30cb56bc49
- sampleSize: 20
- scrollIncrement: 40
- userAgent: Mozilla/5.0 (X11; Linux x86_64) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/42.0.2311.90 Safari/537.36
Metrics:
- gcAmount: gc amount in kbytes
- gcTime: gc time in and ouside of script in ms
- majorGcTime: time of major gcs in ms
- pureScriptTime: script execution time in ms, without gc nor render
- renderTime: render time in and ouside of script in ms
- scriptTime: script execution time in ms, including gc and render

          gcAmount |             gcTime |        majorGcTime |     pureScriptTime |         renderTime |         scriptTime
------------------ | ------------------ | ------------------ | ------------------ | ------------------ | ------------------
          12285.07 |              19.23 |              19.23 |             252.80 |             180.02 |             275.45
>> http://localhost:8002/favicon.ico 0:0 Failed to load resource: the server responded with a status of 404 (Not Found)
              0.00 |               0.00 |               0.00 |             235.17 |             144.47 |             235.59
FBENCHMARK ng1-dart1.x.naive_infinite_scroll
Description:
- appSize: 1
- forceGc: false
- iterationCount: 20
- lang: dart
- regressionSlopeMetric: scriptTime
- runId: 70a9aad0-e78d-11e4-815f-9b30cb56bc49
- sampleSize: 20
- scrollIncrement: 40
- userAgent: Mozilla/5.0 (X11; Linux x86_64) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/42.0.2311.90 Safari/537.36
Metrics:
- gcAmount: gc amount in kbytes
- gcTime: gc time in and ouside of script in ms
- majorGcTime: time of major gcs in ms
- pureScriptTime: script execution time in ms, without gc nor render
- renderTime: render time in and ouside of script in ms
- scriptTime: script execution time in ms, including gc and render

          gcAmount |             gcTime |        majorGcTime |     pureScriptTime |         renderTime |         scriptTime
------------------ | ------------------ | ------------------ | ------------------ | ------------------ | ------------------
          14212.81 |              19.82 |              19.82 |             434.24 |              54.28 |             434.24
>> http://localhost:8002/favicon.ico 0:0 Failed to load resource: the server responded with a status of 404 (Not Found)
FBENCHMARK ng1-dart1.x.naive_infinite_scroll
Description:
- appSize: 2
- forceGc: false
- iterationCount: 20
- lang: dart
- regressionSlopeMetric: scriptTime
- runId: 70a9aad0-e78d-11e4-815f-9b30cb56bc49
- sampleSize: 20
- scrollIncrement: 40
- userAgent: Mozilla/5.0 (X11; Linux x86_64) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/42.0.2311.90 Safari/537.36
Metrics:
- gcAmount: gc amount in kbytes
- gcTime: gc time in and ouside of script in ms
- majorGcTime: time of major gcs in ms
- pureScriptTime: script execution time in ms, without gc nor render
- renderTime: render time in and ouside of script in ms
- scriptTime: script execution time in ms, including gc and render

          gcAmount |             gcTime |        majorGcTime |     pureScriptTime |         renderTime |         scriptTime
------------------ | ------------------ | ------------------ | ------------------ | ------------------ | ------------------
           5671.70 |              15.38 |               0.00 |             476.42 |              63.11 |             491.94
>> http://localhost:8002/favicon.ico 0:0 Failed to load resource: the server responded with a status of 404 (Not Found)
FBENCHMARK ng1-dart1.x.naive_infinite_scroll
Description:
- appSize: 4
- forceGc: false
- iterationCount: 20
- lang: dart
- regressionSlopeMetric: scriptTime
- runId: 70a9aad0-e78d-11e4-815f-9b30cb56bc49
- sampleSize: 20
- scrollIncrement: 40
- userAgent: Mozilla/5.0 (X11; Linux x86_64) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/42.0.2311.90 Safari/537.36
Metrics:
- gcAmount: gc amount in kbytes
- gcTime: gc time in and ouside of script in ms
- majorGcTime: time of major gcs in ms
- pureScriptTime: script execution time in ms, without gc nor render
- renderTime: render time in and ouside of script in ms
- scriptTime: script execution time in ms, including gc and render

          gcAmount |             gcTime |        majorGcTime |     pureScriptTime |         renderTime |         scriptTime
------------------ | ------------------ | ------------------ | ------------------ | ------------------ | ------------------
          12422.22 |              27.16 |               0.00 |             509.40 |              55.74 |             536.69
>> http://localhost:8002/favicon.ico 0:0 Failed to load resource: the server responded with a status of 404 (Not Found)
F
```

**Comment by hterkelsen** [comment]
@tbosch you said you've seen this too?

**Comment by hterkelsen** [comment]
Fixed by #1474
